internal/backend/auth/authjwttokens/jwt.go

```diff
@@ -0,0 +1,91 @@
+package authjwttokens
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	j "github.com/golang-jwt/jwt/v5"
+	"github.com/google/uuid"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/auth/authtokens"
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+const issuer = "ak"
+
+var (
+	method   = j.SigningMethodHS256
+	hashSize = method.Hash.Size()
+)
+
+type Config struct {
+	SignKey string `koanf:"sign_key"`
```

Comment:
this should be mandatory non optional in default mode

Reply:
yup - it is.